lib/inspector.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const assert = require('assert');
+const connect = process.binding('inspector').connect;
+const EventEmitter = require('events');
+
+assert(connect, 'Inspector is not available');
+
+exports.Session = class extends EventEmitter {
```

Comment:
would prefer ...

```js
class Session extends EventEmitter { /** ... **/ }

module.exports = {
  Session
};
```

for consistency with changes made recently in other modules.